pkg/azuredisk/nodeserver_test.go

```diff
@@ -718,14 +718,14 @@ func TestNodeExpandVolume(t *testing.T) {
 
 	invalidPathErr := testutil.TestError{
 		DefaultError: status.Error(codes.NotFound, "failed to determine device path for volumePath [./test]: path \"./test\" does not exist"),
-		WindowsError: status.Errorf(codes.NotFound, "Forward to GetVolumeIDFromTargetPath failed, err=error getting the volume for the mount ./test, internal error error getting volume from mount. cmd: (Get-Item -Path ./test).Target, output: (Get-Item -Path ), error: path \"./test\" does not exist"),
```

Comment:
Please dont revert these changes. They were breaking the windows unit test cases in the last merge.
Same goes for the below changes as well.

Reply:
Which changes should I not revert?

Reply:
Keeping these changes and made some changes to handle the path failures in fake_safe_mounter_windows.go